cmd/dmsg-server/internal/api/api.go

```diff
@@ -0,0 +1,214 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"math"
+	"math/big"
+	"net/http"
+	"time"
+
+	"github.com/go-chi/chi"
+	"github.com/sirupsen/logrus"
+	"github.com/skycoin/dmsg"
+	"github.com/skycoin/dmsg/buildinfo"
+	"github.com/skycoin/dmsg/cipher"
+	"github.com/skycoin/dmsg/httputil"
+	"github.com/skycoin/skycoin/src/util/logging"
+)
+
+// API main object of the server
+type API struct {
+	numberOfClients      int64
+	startedAt            time.Time
+	avgPackagesPerMinute uint64
+	avgPackagesPerSecond uint64
```

Comment:
It seems rather important we include a `sync.Mutex` to avoid read/write race conditions for these variables.

Reply:
Or we can do read/write operations with methods from the `atomic` module.

Reply:
Thank you, Evan! I have added mutex.